versioned/transfer/src/main/java/org/projectnessie/versioned/transfer/ProgressEvent.java

```diff
@@ -28,6 +28,9 @@ public enum ProgressEvent {
   START_COMMITS,
   COMMIT_WRITTEN,
   END_COMMITS,
+  START_GENERIC,
```

Comment:
This value does not appear to be used to produce exports events... should we note that in a comment, perhaps?

Reply:
There's a comment in `org.projectnessie.tools.admin.cli.ExportRepository.ExportProgressListener#progress`